### PR TITLE
Add index.html

### PR DESCRIPTION
**Pull request by ritikarawat220**
HELLO MICROVERSE PROJECT
-Add index.html
-Add style.css
-Linters checked
-README fixed

### CODE REVIEW

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hi @ritikarawat220 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
All linters are well set up and checks passed :heavy_check_mark:
Pull request title is perfect :heavy_check_mark:
Pull request summary is descriptive :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:

## Required Changes ♻️

- I noticed that you deleted the original branch you used for the project.

- **Please Note** that a branch should not be deleted midway through the review process. The initial branch you used as well as the initial pull request for the project is what must be used until the project is approved. Failure to do this can result in your project being invalidated in the future.

- Please make sure to make the requested changes and send back this same Pull request for another review.

- I also noticed that you are missing the  [Microverse Readme template](https://github.com/microverseinc/readme-template). Even though it is in your main branch it has to be added to this branch since that is what you're sending for review. This makes your project incomplete. Every section of the readme template is required with the exception of the FAQ section which is optional. So please make sure to add the README template to your project and fill in all the necessary contents :+1:

- [x] For the template attached, kindly do the following when customizing it to suit this project:

    • Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it :100: 
    • For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. For the Client section, kindly include HTML and CSS :+1: 
    • For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. :wink: 
    • For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
        ◦ Installation of Git
        ◦ Installation of NPM
        ◦ A working browser
        ◦ Internet connection
        ◦ A code editor like Visual Studio Code
        ◦ Installation of Live Server plugging in Visual Studio Code
        ◦ Any other tool you can think about :100: 
    • For the Setup section, kindly add instructions for cloning your repository on a local machine :+1: 
    • For the Install section, kindly add instructions for setting up linters used in this project :+1: 
    • For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
- For the Future Features section include 2 or more future features.
 - Kindly implement all the changes listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student.  


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ritikarawat220** [comment]
> Hi @ritikarawat220 ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!  [ 68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966 ](https://camo.githubusercontent.com/7e9210dcfdfd707fee9896010cd3908fb63e8afc568301ba1c1d5cba19e1499f/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) [ ](https://camo.githubusercontent.com/7e9210dcfdfd707fee9896010cd3908fb63e8afc568301ba1c1d5cba19e1499f/68747470733a2f2f6d65646961312e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)
> 
> ## Highlights
> All linters are well set up and checks passed ✔️ Pull request title is perfect ✔️ Pull request summary is descriptive ✔️ Meaningful commit messages ✔️
> 
> ## Required Changes ♻️
> * I noticed that you deleted the original branch you used for the project.
> * **Please Note** that a branch should not be deleted midway through the review process. The initial branch you used as well as the initial pull request for the project is what must be used until the project is approved. Failure to do this can result in your project being invalidated in the future.
> * Please make sure to make the requested changes and send back this same Pull request for another review.
> * I also noticed that you are missing the  [Microverse Readme template](https://github.com/microverseinc/readme-template). Even though it is in your main branch it has to be added to this branch since that is what you're sending for review. This makes your project incomplete. Every section of the readme template is required with the exception of the FAQ section which is optional. So please make sure to add the README template to your project and fill in all the necessary contents 👍
> * [x]  For the template attached, kindly do the following when customizing it to suit this project:
>   • Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it 💯
>   • For the Tech Stack Section, Kindly remove the Server and Database subsections because the technologies listed there are not relevant to this project. For the Client section, kindly include HTML and CSS 👍
>   • For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 😉
>   • For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
>   ◦ Installation of Git
>   ◦ Installation of NPM
>   ◦ A working browser
>   ◦ Internet connection
>   ◦ A code editor like Visual Studio Code
>   ◦ Installation of Live Server plugging in Visual Studio Code
>   ◦ Any other tool you can think about 💯
>   • For the Setup section, kindly add instructions for cloning your repository on a local machine 👍
>   • For the Install section, kindly add instructions for setting up linters used in this project 👍
>   • For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
> * For the Future Features section include 2 or more future features.
> * Kindly implement all the changes listed above. It will make your README.md file look professional and it will give you an idea of how to set README.md files for most of the projects you will execute as a Microverse student.
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
> 
> You can also consider: N/A
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

thanks for the feedback,
can you check now???

**Comment by nahnahdev** [comment]
Hi @ritikarawat220 
- [ ] You still haven't added the README file to the branch you are working on. You made changes to the README on the `main` branch. Please add the README to your `main-branch` branch instead and follow all my suggestions from before to set it up.
- [ ] Next time please name your branches properly so they don't confuse you. You can read more about this [here](https://gist.github.com/jefffederman/1d492f98b8e3913a75ca)
- [ ] Submit for review on your dashboard when you're done.

File: README.md [hunk 1]
```diff
@@ -1 +1,96 @@
-
+<a name="readme-top"></a>
+<div align="center">
+
+  <img src="./images.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+
+</div>
```
Comment (ABDULALI3468) [inline README.md]: Please consider Removing the Microverse logo. If you have your logo, you can use it but I would advise you to remove it 💯If you want to give complements to Microverse you can use the Acknowledgments section for this purpose.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @ritikarawat220 , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

1- You have removed the following sections from your README file. Please consider adding them as they are crucial enough for the Professional README according to the [Microverse Template](https://github.com/microverseinc/readme-template) and in accordance with the guidelines of the previous Code Reviewer.

> Tech Stack Section
> Prerequisite Section
> Setup section
> Install section
> Usage section

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

1- Kindly re-review your PR title so it describes what you did in PR in a few words.

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ritikarawat220) [reply README.md]: i have removed it.

**Review by osamaashraf6 — APPROVED** [review]
Hi @ritikarawat220 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors present
- The PR has a descriptive PR👍

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**